avahi-common/strlst-test.c

```diff
@@ -49,6 +51,12 @@ int main(AVAHI_GCC_UNUSED int argc, AVAHI_GCC_UNUSED char *argv[]) {
     a = avahi_string_list_add_pair(a, "uxknurz", NULL);
     a = avahi_string_list_add_pair_arbitrary(a, "uxknurz2", (const uint8_t*) "blafasel\0oerks", 14);
     a = avahi_string_list_add(a, "i am a \"string\" with embedded double-quotes (\\\")\nand newlines (\\n).");
+    a = avahi_string_list_add(a, "\x1f");
+    a = avahi_string_list_add(a, "\x20");
+    a = avahi_string_list_add(a, "\x7f");
+    a = avahi_string_list_add(a, "\x80");
+    a = avahi_string_list_add(a, "\xff");
```

Comment:
I think it would be better to put them all in one string like `gh_issue_169=...\xff...` and then look for `gh_issue_169=...`. This way it should be more targeted and for example the other tests that can be added in the future are unlikely to interfere with this particular test.